pages/docs/concepts/asyncapi-document/payload-schema.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Payload Schema
+weight: 239
+---
+
+The payload schema defines a message's format, data types, and properties to ensure that the payload follows a specific structure and data format.
+
+In the AsyncAPI specification, you can use schema languages such as JSON or Avro Schema to create the payload schema. Enabling consumers to gain insights into the structure and data types of the payload.
```

Comment:
```suggestion
In the AsyncAPI specification, you can use schema languages such as JSON Schema or Avro to create the payload schema. Enabling consumers to gain insights into the structure and data types of the payload.
```

Do not split JSON and Schema because the name is JSON Schema. [JSON](https://www.json.org/json-en.html) and [JSON Schema](https://json-schema.org/) are two different things.